### PR TITLE
[Motions 2023 11 cwg 1] P3046R0 Core Language Working Group "ready" Issues

### LINKED ISSUE #6655: [2023-11 CWG Motion 1] Core Language Working Group "ready" Issues for the November, 2023 meeting

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #6655.

Notes:
* CWG1038 is fixed by CWG2725
* CWG1698 is fixed by CWG2747

Fixes cplusplus/CWG#320
Fixes cplusplus/CWG#22
Fixes cplusplus/CWG#33
Fixes cplusplus/CWG#54
Fixes cplusplus/CWG#63
Fixes cplusplus/CWG#69
Fixes cplusplus/CWG#143
Fixes cplusplus/CWG#325
Fixes cplusplus/CWG#348
Fixes cplusplus/CWG#349
Fixes cplusplus/CWG#356
Fixes cplusplus/CWG#354
Fixes cplusplus/CWG#358
Fixes cplusplus/CWG#355
Fixes cplusplus/CWG#391
Fixes cplusplus/CWG#410
Fixes cplusplus/CWG#436
Fixes cplusplus/CWG#420
Fixes cplusplus/CWG#198

Fixes cplusplus/papers#1585

### CODE REVIEW

File: source/expressions.tex [hunk 1]
```diff
@@ -6576,8 +6603,10 @@
 Otherwise, the result of each of the operators is unspecified.
 \begin{note}
 A relational operator applied
-to unequal function pointers or to unequal pointers to \tcode{void}
+to unequal function pointers
 yields an unspecified result.
+A pointer value of type ``pointer to \cv void''
```
Comment (JohelEGP) [inline source/expressions.tex:6608]:
```suggestion
A pointer value of type ``pointer to \cv{}~\keyword{void}''
```

File: source/expressions.tex [hunk 2]
```diff
@@ -5048,31 +5057,29 @@
...
-exception\iref{except.throw}.
 
 \begin{bnf}
 \nontermdef{noexcept-expression}\br
   \keyword{noexcept} \terminal{(} expression \terminal{)}
 \end{bnf}
 
+\pnum
+The operand of the \keyword{noexcept} operator
+is an unevaluated operand\iref{expr.context}.
```
Comment (JohelEGP) [inline source/expressions.tex:5070]:
```suggestion
is an unevaluated operand\iref{term.unevaluated.operand}.
```

File: source/expressions.tex [hunk 3]
```diff
@@ -4556,7 +4561,9 @@
 Its operand shall be a prvalue of type ``pointer to \tcode{T}'',
 where \tcode{T} is an object or function type.
 The operator yields an lvalue of type \tcode{T}
```
Comment (JohelEGP) [inline source/expressions.tex:4563]:
```suggestion
The operator yields an lvalue of type \tcode{T}.
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

File: source/expressions.tex (hunk 1 above)
Comment (burblebee) [reply source/expressions.tex:6608]: thanks!

File: source/expressions.tex (hunk 2 above)
Comment (burblebee) [reply source/expressions.tex:5070]: Added in separate commit.

**Review by JohelEGP — APPROVED** [review]

**Pull request merged**